src/modelinstance.cpp

```diff
@@ -781,6 +781,17 @@ Status ModelInstance::fetchModelFilepaths() {
             modelFiles.push_back(file);
         }
     }
+    if (!found) {
+        found = true;
+        modelFiles.clear();
+        for (auto extension : TF_MODEL_FILES_EXTENSIONS) {
+            auto file = findModelFilePathWithExtension(extension);
+            if (file.empty()) {
+                found = false;
+            }
+            modelFiles.push_back(file);
+        }
+    }
```

Comment:
Write function/macro to unify code with PDPD, ONNX, OV extensions. Then it will be clear that this function mostly defines the order of checking